src/components/socialLogin/FacebookConfig.js

```diff
@@ -0,0 +1,2 @@
+export const AppId = '404512643718985';
```

Comment:
Move this to a separate config file and use environment variables instead. Do not hardcode secret values

Reply:
@adesege worked on feedback